boltdesignsystem/bolt
# DS-887: Remove ja/en lang logic

## Jira

https://pegadigitalit.atlassian.net/browse/DS-887

## Summary

Remove Japanese/English language logic from Bolt.

## Details

Bolt used to output separate files for Japanese and English webpages. These files controlled the spacing and font-families used for each. There were also configuration options for the webpack build that handled languages. All of this is no longer used as language differences are [handled by CSS vars](https://github.com/boltdesignsystem/bolt/blob/master/packages/global/styles/00-vars/_vars-japanese.scss).

This PR removes all remnants of the original language logic from Bolt. It leaves the CSS vars in place.

## How to test

- Review code.
- Run this branch locally and verify the Japanese homepage template looks the same [on prod](https://boltdesignsystem.com/pattern-lab/patterns/999-archive-homepage-homepage-japanese/999-archive-homepage-homepage-japanese.html) as it does locally.

## Release notes

- The `$bolt-lang` SCSS variable has been removed.
- The `lang` and `i18n` boltrc config options have been removed.
- None of these options are needed anymore.
